File: src/2235-Add-Two-Integers/add_two_integers.cpp
#include <iostream>

class Solution {
public:
    int sum(int num1, int num2) {
        return num1 + num2;
    }
};

int main() {
  int num1 = 12, num2 = 5;
  int solution = Solution().sum(num1, num2);
  std::cout << solution;
  
  return 0;
}

File: src/2235-Add-Two-Integers/go/add_two_integers.go
package main

import "fmt"

func sum(num1 int, num2 int) int {
	res := num1 + num2
	return res
}

func main() {
	num1 := 12
	num2 := 5

	result := sum(num1, num2)
	fmt.Println(result)
}
